app/assets/javascripts/angular/controllers/static_pages.js.erb

```diff
@@ -15,4 +15,16 @@ app.controller('StaticPagesCtrl', ['$scope', '$http', '$auth', '$location', func
     });
   }
 
+  var orderBy = $filter('orderBy');
```

Comment:
<img width="883" alt="screen shot 2015-09-14 at 3 43 39 pm" src="https://cloud.githubusercontent.com/assets/13043141/9844643/6dca5d18-5af7-11e5-8139-78cc2d3c0947.png">
This way looks a bit clearer to me and will probably be easier if we are trying to add more sorting options in the future.